app/unit-tests/src/org/commcare/android/util/TestUtils.java

```diff
@@ -362,12 +363,19 @@ private static EvaluationContext buildContextWithInstances(UserSandbox sandbox,
         return new EvaluationContext(null, instances);
     }
 
-    public static void assertFormValue(String expr, Object expectedValue){
+    public static void assertFormValue(String expr, Object expectedValue) {
         FormDef formDef = FormEntryActivity.mFormController.getFormEntryController().getModel().getForm();
         FormInstance instance = formDef.getMainInstance();
 
         String errorMsg;
         errorMsg = ExprEvalUtils.expectedEval(expr, instance, null, expectedValue, null);
         assertTrue(errorMsg, "".equals(errorMsg));
     }
+
+    public static Object getFormValue(String expr) throws XPathSyntaxException {
+        FormDef formDef = FormEntryActivity.mFormController.getFormEntryController().getModel().getForm();
+        FormInstance instance = formDef.getMainInstance();
+        EvaluationContext ec = new EvaluationContext(instance);
+        return ExprEvalUtils.xpathEval(ec, expr);
+    }
```

Comment:
@shubham1g5 I have created this method to retrieve a value from a form question, but I wonder if there isn't a method to do this already.